src/js/model/RacingCar.js

```diff
@@ -0,0 +1,41 @@
+class RacingCar {
+  constructor(cars = [], count, record = {}, winners = []) {
+    this.cars = cars;
+    this.count = count;
+    this.record = record;
+    this.winners = winners;
+  }
```

Comment:
```suggestion
class RacingCar {
  #cars;
  #count;
  #record;
  #winners;
    
  constructor(cars = [], count, record = {}, winners = []) {
    this.#cars = cars;
    this.#count = count;
    this.#record = record;
    this.#winners = winners;
  }
```
이렇게 private으로 선언하면 어떨까요?